proteinworkshop/datasets/base.py

```diff
@@ -418,28 +418,29 @@ def process(self):
         """
         if not self.overwrite:
             if self.chains is not None:
-                pdb_codes = [
+                index_pdb_tuples = [
                     (i, pdb)
                     for i, pdb in enumerate(self.pdb_codes)
                     if not os.path.exists(
                         Path(self.processed_dir) / f"{pdb}_{self.chains[i]}.pt"
                     )
                 ]
             else:
-                pdb_codes = [
+                index_pdb_tuples = [
                     (i, pdb)
                     for i, pdb in enumerate(self.pdb_codes)
                     if not os.path.exists(
                         Path(self.processed_dir) / f"{pdb}.pt"
                     )
                 ]
-            logger.info(f"Processing {len(pdb_codes)} unprocessed structures")
+            logger.info(f"Processing {len(index_pdb_tuples)} unprocessed structures")
         else:
-            pdb_codes = self.pdb_codes
+            index_pdb_tuples = self.pdb_codes
 
         raw_dir = Path(self.raw_dir)
-        for i, pdb in enumerate(tqdm(pdb_codes)):
+        for index_pdb_tuple in tqdm(index_pdb_tuples):
```

Comment:
I don't think it holds that `self.pdb_codes` will be a tuple (as assumed in L438)

Reply:
Great catch! I'll fix this now.

Reply:
I can confirm that I can train PPI models after this [latest fix](https://github.com/a-r-j/ProteinWorkshop/pull/52/commits/41601c4aaef20dea38463aa14040bf519fd9efb3). Seeing validation F1 scores of around 0.86 after 10 epochs of training with schnet.